Docs/source/usage/workflows/ml_dataset_training.rst

```diff
@@ -0,0 +1,254 @@
+.. _ml_dataset_training:
+
+Training a surrogate model from WarpX data
+==========================================
+
+Suppose we have a WarpX simulation that we wish to replace with a neural network surrogate model.
+For example, a simulation determined by the following input script
+
+.. dropdown:: Python Input for Training Simulation
+    :color: light
+    :icon: info
+    :animate: fade-in-slide-down
+
+    .. literalinclude:: ml_materials/run_warpx_training.py
+    :language: python3
+
+In this section we walk through a workflow for data processing and model training.
+This workflow was developed and first presented in (CITE PASC submission).
+
+This assumes you have an up-to-date environment with PyTorch and openPMD.
+
+Data Cleaning
+-------------
+
+It is important to inspect the data for artifacts to
+check that input/output data make sense.
+If we plot the final phase space for beams 1-8,
+the particle data is distributed in a single blob.
+
+.. figure:: ml_materials/stage_1_final_beam.png
+   :alt: [fig:stage_1_final_beam] Plot comparing model prediction with simulation output.
+
+   [fig:stage_1_final_beam] Plot showing the final phase space projections of the training data for stage 1.
+
+This is as we expect and what is optimal for training neural networks.
+On the other hand, the final phase space for beam 0 has a halo of outlying particles.
+
+.. figure:: ml_materials/stage_0_final_beam.png
```

Comment:
Please add the PNGs in the PR description and use the link to them (do not add to `git` to keep the WarpX repo small :) )

Reply:
done!